<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicoForm</class>
 <widget class="QWidget" name="PicoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>165</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="binlab">
       <property name="text">
        <string>BinDir</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="binDir">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="binflab">
       <property name="text">
        <string>BinFile</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="binFile">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="picodirlab">
       <property name="text">
        <string>PicoDir</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="picoDir">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="picolab">
       <property name="text">
        <string>PicoPort</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="portSel">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="picoPort">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="autoDl">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Download automatically after reset when binary is avail.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Auto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionViewBin">
   <property name="icon">
    <iconset resource="../../../../Qt/qt-library/lib/utils/stdicons.qrc">
     <normaloff>:/stdicons/eye.png</normaloff>:/stdicons/eye.png</iconset>
   </property>
   <property name="text">
    <string>ViewBin</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;View the Bin (build) directory in file manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSelBin">
   <property name="icon">
    <iconset resource="../../../../Qt/qt-library/lib/utils/stdicons.qrc">
     <normaloff>:/fileopen</normaloff>:/fileopen</iconset>
   </property>
   <property name="text">
    <string>SelBin</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the Bin (build) directory in file manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSelPicoDir">
   <property name="icon">
    <iconset resource="rsc/rsc.qrc">
     <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
   </property>
   <property name="text">
    <string>SelPicoDir</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the directory where the Pico will be mounted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="../../../../Qt/qt-library/lib/utils/stdicons.qrc">
     <normaloff>:/bottom</normaloff>:/bottom</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Download the binary to Pico</string>
   </property>
  </action>
  <action name="actiondelBin">
   <property name="icon">
    <iconset resource="../../../../Qt/qt-library/lib/utils/stdicons.qrc">
     <normaloff>:/cut</normaloff>:/cut</iconset>
   </property>
   <property name="text">
    <string>delBin</string>
   </property>
  </action>
  <action name="actionBoot">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../Qt/qt-library/lib/utils/stdicons.qrc">
     <normaloff>:/stdicons/launch.svgz</normaloff>:/stdicons/launch.svgz</iconset>
   </property>
   <property name="text">
    <string>Boot</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Boot target CPU&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsc/rsc.qrc"/>
  <include location="../../../../Qt/qt-library/lib/utils/stdicons.qrc"/>
 </resources>
 <connections/>
</ui>
